test/TestProjects/ResourceClients-LowLevel/Generated/ResourceGroup.cs

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace ResourceClients_LowLevel
+{
+    /// <summary> The ResourceGroup service client. </summary>
+    public partial class ResourceGroup
+    {
+        private const string AuthorizationHeader = "Fake-Subscription-Key";
+        private readonly AzureKeyCredential _keyCredential;
+        private readonly HttpPipeline _pipeline;
+        private readonly ClientDiagnostics _clientDiagnostics;
+        private readonly string _groupId;
+        private readonly Uri _endpoint;
+
+        /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
+        public virtual HttpPipeline Pipeline => _pipeline;
+
```

Comment:
There should be a public GroupId property here, per: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-subclient-properties